Sprint-2/implement/bmi.js

```diff
@@ -13,3 +13,15 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+
+function CalculateBMI (weight, height){
```

Comment:
The convention in JS is to write functions in camelCase. calculateBMI

Classes and React components are written in PascalCase.